### ISSUE TITLE
TypeError: pivot() takes 1 positional argument but 4 were given

### ISSUE THREAD

**Issue by Karenina-na** [MarvinT/calmap#17]
Hello, I have encountered some issues while using the function `calmap.calendarplot.`
Here is my code:
```python
from doc.Plot.require import *
import calmap

# Import Data
df = pd.read_csv("https://raw.githubusercontent.com/selva86/datasets/master/yahoo.csv", parse_dates=['date'])
# df = pd.read_csv("../data/yahoo.csv", parse_dates=['date'])
df.set_index('date', inplace=True)

# Plot
plt.figure(figsize=(16, 10), dpi=60)
calmap.calendarplot(df[df['year'] == 2014]['VIX.Close'], fig_kws={'figsize': (16, 10)}, how="sum",
yearlabel_kws={'color': 'black', 'fontsize': 14}, subplot_kws={'title': 'Yahoo Stock Prices'})
plt.show()
```

```markdown
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
Cell In[49], line 11
      9 # Plot
     10 plt.figure(figsize=(16, 10), dpi=60)
---> 11 calmap.calendarplot(df[df['year'] == 2014]['VIX.Close'], fig_kws={'figsize': (16, 10)}, how="sum",
     12 yearlabel_kws={'color': 'black', 'fontsize': 14}, subplot_kws={'title': 'Yahoo Stock Prices'})
     13 plt.show()

File ~\AppData\Roaming\Python\Python39\site-packages\calmap\__init__.py:403, in calendarplot(data, how, yearlabels, yearascending, yearlabel_kws, subplot_kws, gridspec_kws, fig_kws, fig_suptitle, vmin, vmax, **kwargs)
    400 max_weeks = 0
    402 for year, ax in zip(years, axes):
--> 403     yearplot(by_day, year=year, how=None, ax=ax, **kwargs)
    404     max_weeks = max(max_weeks, ax.get_xlim()[1])
    406     if yearlabels:

File ~\AppData\Roaming\Python\Python39\site-packages\calmap\__init__.py:202, in yearplot(data, year, how, vmin, vmax, cmap, fillcolor, linewidth, linecolor, daylabels, dayticks, monthlabels, monthticks, monthly_border, ax, **kwargs)
    197 by_day.loc[(by_day.index.month == 12) & (by_day.week < 10), "week"] = (
    198     by_day.week.max() + 1
    199 )
    201 # Pivot data on day and week and mask NaN days. (we can also mask the days with 0 counts)
--> 202 plot_data = by_day.pivot("day", "week", "data").values[::-1]
    203 plot_data = np.ma.masked_where(np.isnan(plot_data), plot_data)
    205 # Do the same for all days of the year, not just those we have data for.

TypeError: pivot() takes 1 positional argument but 4 were given
```

and the variables `df[df['year'] == 2014]['VIX.Close']` output is

```markdown
df[df['year'] == 2014]['VIX.Close'] output:

date
2014-01-02    14.230000
2014-01-03    13.760000
2014-01-06    13.550000
2014-01-07    12.920000
2014-01-08    12.870000
                ...    
2014-12-24    14.370000
2014-12-26    14.500000
2014-12-29    15.060000
2014-12-30    15.920000
2014-12-31    19.200001
```

I am uncertain about how to modify this code. Even when I search for examples on Stack Overflow that appear identical, they still result in errors. Could you please guide me on how to modify this code? Thank you.

**Comment by hugovk** [comment]
Will be fixed by #14 or #15.

**Comment by MarvinT** [comment]
should be resolved with newly released version

**Comment by hugovk** [comment]
Thank you, calmap 0.0.11 is now working with Pandas 2!

Please could you push a Git tag for the released version 0.0.11? And also optionally create a matching [GitHub release](https://github.com/MarvinT/calmap/releases)?
